arbitrum-docs/build-decentralized-apps/how-tos/migrate-nfts-arbitrum.mdx

```diff
@@ -0,0 +1,137 @@
+---
+id: migrate-nfts-arbitrum
+title: "How to migrate your NFT collection to Arbitrum"
+sidebar_label: "Migrate your NFT collection to Arbitrum"
+description: Learn how to migrate your NFT collection to Arbitrum through third-party bridges and DIY bridging solutions.
+author: amarrazza
+content-type: how-to
+reader_audience: Developers who want to migrate their NFT collections to Arbitrum from either Ethereum mainnet or other non-EVM chains.
+---
+
+
+import PublicPreviewBannerPartial from '../../partials/_public-preview-banner-partial.md'; 
+
+<PublicPreviewBannerPartial />
+
+This how-to is for **developers** who want to migrate existing <a data-quicklook-from="non-fungible-tokens-nfts">NFT</a> collections to <a data-quicklook-from="arbitrum-one">Arbitrum One</a> or <a data-quicklook-from="arbitrum-nova">Nova</a>.
+
+This guidance applies to NFT collections used within web3 games, decentralized apps (dApps), and other use-cases.
+
+### Prerequisites
+
+ - An NFT collection that's minted and managed by smart contracts on either an **EVM-compatible** chain (like Ethereum mainnet) or a supported **non-EVM** chain[^1].
+ - NFTs that are implemented using the ERC-721 or ERC-1155 token standards[^2]. (todo - clarify how this prereq can be satisfied for non-EVM chains)
```

Comment:
I wouldn't get into the details of these historical ERC-20 use cases (i.e., I would remove the footnote). I don't think it adds value to the audience this page is intended to, imo.